samples/KristofferStrube.Blazor.FileSystemAccess.WasmExample/Program.cs

```diff
@@ -10,7 +11,14 @@
 builder.RootComponents.Add<HeadOutlet>("head::after");
 
 builder.Services.AddScoped(sp => new HttpClient { BaseAddress = new Uri(builder.HostEnvironment.BaseAddress) });
-builder.Services.AddFileSystemAccessServiceInProcess();
+
+// Configure with custom script path
+builder.Services.AddFileSystemAccessServiceInProcess(options =>
+{
+    // The file at this path in this example is manually copied to wwwroot folder
+    options.ScriptPath = $"/content/custom-path/{FileSystemAccessOptions.DefaultNamespace}.js";
```

Comment:
I see the point in showing the example of using the option, but I don't think most will do this. So I would add this manually copied script. You can keep this line, but comment it out so that people see that this is possible.